### DISCUSSION TITLE
2023/08/art-exhibition/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#88] [General]
# 2023/08/art-exhibition/

https://xiangyun.rbind.io/2023/08/art-exhibition/

**Comment by earfanfan** [comment]
我有一个问题，对待不同形式、不同价值的艺术作品应该抱着什么心态去欣赏呢？

越少见的就该用那种捧着它、珍视它的心态去看呢，还是将一切都视如寻常，也许陈列在博物馆、美术馆的一幅画很珍贵，传承多年保存起来不容易，看到的机会很少是该珍惜这种看到的机会，但是生活中能看到一只蝴蝶振翅的机会也很少啊，这两种少都该视作少么

**Reply by XiangyunHuang** [reply]
我不知道一般的情况，我是抱着学习的心态去的，想去看原作。顺其自然就好，不喜欢的风格就跳着看。

能在旅游旺季陈列在国家博物馆、中国美术馆的都很珍贵。国家大剧院最近展览东京富士美术馆藏西方人物绘画作品，收费的，它就直接提醒观众，如果有人弄坏了，就照价赔偿，追究法律责任，这没什么说的。

看蝴蝶振翅和看画，各有所好。

**Reply by earfanfan** [reply]
貌似那些外国的国家级博物馆的一些展品来我们国家展出的话，只会在北京或上海。记得前几年有一个大英博物馆的一个展去了上海，当时想去看，但是没去。疫情这几年，撒撒都断了，不过最近也没关注过，所以不了解了。

**Reply by XiangyunHuang** [reply]
国家大剧院的这个西方绘画展，我感觉内容和中国美术馆有大不同，我也买了个票，有空的时候，提前预约就可以去。我买了一本专门介绍大英博物馆的馆藏作品的书《英国国家博物馆·不列颠的馆藏》，在王府井书店，看到锁在柜子里，定价 1200多，不买不给看，我就去网上买了，相比而言，挺划算的，差不多打五折了。我感觉这类书买的人应该比较少，锁在柜子里，和导购商量也不给看，挺有意思的，不知道他们要卖给谁，可能是为了装饰书架，显得很有品味。

**Reply by XiangyunHuang** [reply]
我感觉我不要再买书了，现在这几本书至少够我看一年半载了，还要跑跑美术馆，偶尔写点观后感，时间都安排上了。

**Reply by earfanfan** [reply]
讲真，你在买书这件事上好舍得花钱啊，你提到的那些画册还有大英博物馆这几本我都蛮想看的，猴年马月有机会的话借给我看看。我也是打算先把买了的看完再说。

**Reply by XiangyunHuang** [reply]
等猴年马月再说这个态度比较好，不要特意为了别人的宣传去强行买买买，我还想观望一下自己是临时短期的热情还是想把它当作长期的业余爱好。对我来说，梵高、莫奈的画册加上《英国国家博物馆·不列颠的馆藏》一共三本，足够了，我想把它们作为移情的对象。

**Reply by earfanfan** [reply]
我说的猴年马月是指，等你看完了寄给我看……为撒要移情，，转移对于看画的热情么，不叨了，俺干活去也

**Reply by XiangyunHuang** [reply]
有时候工作感到身心俱疲，看画看画册都是移情。

**Reply by XiangyunHuang** [reply]
> 我买了一本专门介绍大英博物馆的馆藏作品的书《英国国家博物馆·不列颠的馆藏》

今天书到手了，我发现我搞错了，英国大英博物馆相当于咱们的国家博物馆，我买的书是关于英国国家美术馆的，相当于咱们的中国国家美术馆。书很厚很大，印刷质量很高，超出想象的厚和大，我都怀疑是不是之前在王府井书店看错了，它高高地收在玻璃柜里，可能当时发生了视觉上的错觉。

**Reply by earfanfan** [reply]
哦，里面是讲撒的，我以前看过贝聿铭的作品集，里面有没撒好看的建筑或雕塑。话又说回来，我好像受到了你的无形安利，昨天开始看局部的第一季，你绝对猜不到我被打动的点是撒，哈哈哈。

超出想象的厚和大是多厚多大啊。

**Reply by XiangyunHuang** [reply]
> 超出想象的厚和大是多厚多大啊。

285 x 405 毫米的，10余斤，500 多页囊括欧洲各主要国家的一些作品。

**Reply by XiangyunHuang** [reply]
> 昨天开始看局部的第一季，你绝对猜不到我被打动的点是撒，哈哈哈。

那我就不猜了，你说吧。

**Comment by XiangyunHuang** [comment]
第二次再去的时候，我不该赶着去看，不该想着在给定的时间内看完。我突然找不到第一次的看画感觉了，那种一幅之后又一幅惊艳到我的新鲜感。我不知道后面还有多少，那种对未知的期待，对新鲜的追求得到满足的快感。这次看完之后，对下次充满期待的感觉。几个月之后，我还会再去看的，看还能不能找到那种感觉。我取消了紧接着的第三次的预约。八大山人、石涛系列展，以及往前的文人画、山水画都看不懂了，只一些葡萄、杏呀，留下了挺深的印象，隔着几百年，还是透亮、新鲜。

来看《时代的容颜》的展览，没有小孩了，可能是收费，也可能是西方人物画中，女子坦胸露背的比较多。清静了，安静了，蛮好。每一个女子肤如凝脂，丰润，那些小孩的面庞太美，陈丹青说画得雌雄同体了。惊讶到我的是油画可以将人物细节画得那样逼真，衣服上的图案，蕾丝边的褶皱都很细腻。

**Reply by earfanfan** [reply]
>我突然找不到第一次的看画感觉了……

找感觉这种事是很玄妙的。理论上来说，如果你真心喜欢某一幅画，可能每次看到都会很有感觉，或者常看常新，又悟出新的体会来。假设你第一次看画是将积攒多年的需求一次性释放，那后面看画情绪波动幅度降低、感觉相对变得平淡也很正常啊。这看你是否是真心喜欢，还是追逐心流体验。

话又说回来，你提到的《时代的容颜》我好有兴趣，要是有拍下来一些的话，介不介意微信发给我瞅瞅？
